misc/config_tools/schema/config.xsd

```diff
@@ -129,6 +129,14 @@ If your VM is not a security VM, leave this option unchecked. </xs:documentation
         <xs:documentation>Configure Software SRAM. This feature reserves memory buffers as always-cached memory to improve an application's real-time performance.</xs:documentation>
       </xs:annotation>
     </xs:element>
+    <xs:element name="CPU_PERFORMANCE_POLICY" type="policyType" default="Performance">
+      <xs:annotation acrn:title="CPU performance policy type" acrn:views="advanced">
+        <xs:documentation>Select the default performance policy type for all CPUs. The hypervisor will setup CPU frequency based on the policy type.
+
+- ``Performance``: CPU runs at its maximum frequency. Enable hardware autonomous frequency selection if HWP is presented.
```

Comment:
Change "if HWP is presented" to "if the system supports [full term] (HWP)". Replace [full term] with the term that HWP stands for (is it Hardware P-state?).